cypress/support/commands.ts

```diff
@@ -507,6 +513,9 @@ export const createTelegraf = (
     },
   })
 }
+/*
+// Did not find where this is used
+// Replaced with more general command below
```

Comment:
I'm fine with removing this function altogether, also couldn't find any usages

Reply:
Done